relative-path/src/lib.rs

```diff
@@ -290,26 +290,48 @@
 
 #![allow(clippy::manual_let_else)]
 #![deny(missing_docs)]
+#![no_std]
 
+#[cfg(feature = "alloc")]
+extern crate alloc;
+
+#[cfg(feature = "std")]
+extern crate std;
+
+#[cfg(feature = "std")]
+use std::prelude::v1::*;
```

Comment:
Please expand these imports. They include things from alloc (Box, String, traits, ...) and it makes it hard to keep track of what we depend on exactly. For tests I don't really care.

Reply:
there was actually nothing using anything from the prelude :-)